### PR TITLE
Changes for #979 - Document Prereq's

### PR DESCRIPTION
**Pull request by jasnow**
Changes for #979: Document Prereq's

### Test plan

See included automated tests.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -99,6 +100,25 @@ Early in our project we've defined some guidelines for how working with sorbet s
...
+#### Linux: Sorbet tests on Ubuntu 18 but expect it to work on
+almost everything using i686i.
+
+#### Mac OS: We test on OSX Mojave but we publish gems from Darwin
+version 14 (Yosemite) to 18 (Mojave) so we expect it to work on
+the older versions since it is a statically linked binary.
+
+#### Windows: We do not work on Windows so we do not have test
+experience on this platform. We welcome community help with this.
+
```
Comment (jez) [inline README.md]: The README is intended for contributors, but you documented things from the perspective of a Sorbet user.

The project prerequisites for building Sorbet are already documented in the `Quickstart` section of the README.

We use sorbet.org to document things that users would like to know about Sorbet. I've moved these changes to the FAQ.

**Comment by jasnow** [comment]
@jez - Thanks - reason I did not select FAQ was the other FAQ item were language-specific, not environment. Fine with it in FAQ. Thanks again.

**Comment by DarkDimius** [comment]
@jasnow thank you for your contribution!

**Comment by jasnow** [comment]
@jez - You might want to specify which Ruby, such as Ruby MRI, in the FAQ.

**Pull request merged**